core_impl/src/test/java/com/google/instrumentation/stats/MeasurementDescriptorToViewMapTest.java

```diff
@@ -31,12 +32,11 @@
 
   @Test
   public void testPutAndGetView() {
-    View expected = DistributionView.create(
+    MutableView expected = MutableDistributionView.create(
         RpcConstants.RPC_CLIENT_ROUNDTRIP_LATENCY_VIEW,
-        null,
-        Timestamp.fromMillis(System.currentTimeMillis()),
+        new ArrayList<MutableDistribution>(),
         Timestamp.fromMillis(System.currentTimeMillis()));
-    measurementDescriptorToViewMap.putView(
+    measurementDescriptorToViewMap.putMutableView(
```

Comment:
putMutableView is a mouthful. is elaborating this in the name very important, or can we rely on the type system to explain?